python-api-examples/non_streaming_server.py

```diff
@@ -862,6 +876,7 @@ def create_recognizer(args) -> sherpa_onnx.OfflineRecognizer:
             max_active_paths=args.max_active_paths,
             hotwords_file=args.hotwords_file,
             hotwords_score=args.hotwords_score,
+            blank_penalty=args.blank_penalty,
```

Comment:
Could you also update
https://github.com/k2-fsa/sherpa-onnx/blob/b9291246e86f6f4aff68449cd5a228feb2d6695d/sherpa-onnx/python/sherpa_onnx/offline_recognizer.py#L38
to add an extra argument
```
blank_penalty: float = 0
```
and also add docstring for it?

Note that you need to pass the argument to
https://github.com/k2-fsa/sherpa-onnx/blob/b9291246e86f6f4aff68449cd5a228feb2d6695d/sherpa-onnx/python/sherpa_onnx/offline_recognizer.py#L114

Reply:
Fixed! 
I also noticed there is no docstring for the hotwords params, as well as the max_active_paths was not propagated into the `OfflineRecognizerConfig`. Should I raise two seperate PR to fix these?

Reply:
> Should I raise two seperate PR to fix these?

Yes, please do that in a separate PR. Thanks!